docker/generic/start_proxy.py

```diff
@@ -409,6 +409,25 @@ def make_argparser():
         Set the retry times for service control Report request.
         Must be >= 0 and the default is 5 if not set.
         ''')
+    parser.add_argument(
+        '--backend_retry_ons',
+        default=None,
+        help='''
+        The conditions under which ESPv2 does retry on the backends. One or more
+        retryOn conditions can be specified by comma-separated list. 
+        The default is `reset,connect-failure,refused-stream`. 
+        
+        All the retryOn conditions are defined in the 
+        x-envoy-retry-on(https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-on) and 
+        x-envoy-retry-grpc-on(https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-on).
+        ''')
+    parser.add_argument(
+        '--backend_retry_num',
+        default=None,
+        help='''
+        The allowed number of retries. Must be >= 0 and defaults to 1. 
+        If backend_retry_ons is not set, retry_num is effectless.
```

Comment:
remove line 429 since we have default for "backend_retry_ons"

Reply:
Done.